specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2023-06-28-preview/managedccf.json

```diff
@@ -0,0 +1,689 @@
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedCCFNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the Managed CCF is deleted."
+          },
+          "202": {
+            "description": "The Managed CCF resource will be deleted asynchronously"
+          },
+          "204": {
+            "description": "The request was successful; Managed CCF does not exist."
+          },
+          "default": {
+            "description": "Error response describing the reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ConfidentialLedgerDelete": {
+            "$ref": "./examples/ManagedCCF_Delete.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Managed CCF"
+        ],
+        "summary": "Creates a Managed CCF.",
+        "description": "Creates a Managed CCF with the specified Managed CCF parameters.",
+        "operationId": "ManagedCCF_Create",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedCCFNameParameter"
+          },
+          {
+            "name": "ManagedCCF",
+            "in": "body",
+            "description": "Managed CCF Create Request Body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagedCCF"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful and ledger was created; response contains the Managed CCF",
+            "schema": {
+              "$ref": "#/definitions/ManagedCCF"
+            }
+          },
+          "201": {
+            "description": "The Managed CCF resource will be created asynchronously"
```

Comment:
>asynchronously [](http://example.com/codeflow?start=69&length=14)

doc completeness/user-friendliness - consider mentioning the need to check provisionoingState to monitor the progress

Reply:
Added!